### ISSUE TITLE
Game crash on Linux

### ISSUE THREAD

**Issue by sjurske** [Librelancer/Librelancer#82]
Game crashes when I want to land on Planet Pittsburgh.
Crash output:

```
Librelancer has crashed. See the log for more information.

One or more errors occurred. (Queue empty.)
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at LibreLancer.CGameSession.HandlePacket(IPacket pkt)
   at LibreLancer.CGameSession.UpdatePackets()
   at LibreLancer.SpaceGameplay.Update(Double delta)
   at LibreLancer.FreelancerGame.Update(Double elapsed)
   at LibreLancer.Game.Run()
   at lancer.MainClass.<>c__DisplayClass0_0.<Main>b__0()
   at LibreLancer.AppHandler.Run(Action action, Action onCrash)
Inner Exception #1:
Queue empty.
   at System.Collections.Generic.Queue`1.ThrowForEmptyQueue()
   at System.Collections.Generic.Queue`1.Dequeue()
   at LibreLancer.Media.AudioManager.CreateStreaming(StreamingSound sound, String info)
   at LibreLancer.Media.MusicPlayer.Play(String filename, Single attenuation, Boolean loop)
   at LibreLancer.SoundManager.PlayMusic(String name, Boolean oneshot)
   at LibreLancer.RoomGameplay.SwitchToRoom(Boolean dolanding)
   at LibreLancer.RoomGameplay..ctor(FreelancerGame g, CGameSession session, String newBase, BaseRoom room, String virtualRoom)
   at LibreLancer.CGameSession.SceneChangeRequired()
   at LibreLancer.CGameSession.LibreLancer.Net.IClientPlayer.BaseEnter(String _base, String[] rtcs, NewsArticle[] news, SoldGood[] goods, NetSoldShip[] ships)
   at LibreLancer.Net.GeneratedProtocol.HandleClientPacket(IPacket pkt, IClientPlayer player, INetResponder res)

Librelancer Version: fefd720-git (20220415)
```

**Comment by CallumDev** [comment]
This took me a while to reproduce since my system wasn't bringing this issue up, but it will be fixed with the timing work done in the server-auth branch. Will close this issue when it's merged

**Comment by CallumDev** [comment]
Issue fixed in 0555f3a. Re-open if you encounter it again.
